docs/star-tool.md

```diff
@@ -0,0 +1,226 @@
+
+```
+star reload C:\databases\mydb C:\databases\myDatabase.sqlite3
+```
+
+*Note: only files created with the `unload` command can be reloaded.*
+
+##### `unload`
+
+```
+star unload [options] <db> <file>
+```
+
+The `unload` command selects the SQLite database file at the path given in the `<file>` argument, or creates one if it doesn't exists, and then takes data from a database referenced by the absolute or relative path given in the `<db>` argument and imports that data into the created SQLite database. To load data from an existing database located at `C:\databases\mydb` into a new SQLite database file named `myDatabase.sqlite3` at `C:\databases`, we run:
+
+```
+star unload C:\databases\mydb C:\databases\myDatabase.sqlite3
+```
+
+#### Information
+
+##### `list`
+
+```
+star list [options] [command]
+```
+
+The `list` command is used together with one of the following sub-commands to list ranges of entities in a database:
+
+###### `list table`
+
+```
+star list table [options] <db>
+```
+
+The `list table` sub-command lists all tables in the database referenced by the absolute or relative path given in the `<db>` argument. The following options are available to configure the output of the commmand:
+
+Option     | Short form | Description
+---------- | ---------- | ----------------------------------------------------------------------------------------
+`--user`   | `-u`       | Includes user tables in the output.
+`--system` | `-s`       | Includes system tables in the output.
+`--format` | `-f`       | Lets you configure the formatting of the output by providing one or more format options.
+
+Formatting options:
+
+- `t` – The name of the table to list.
+- `b` – The name of the base table (if inherited) of the table to list.
+
+To list all tables of an existing database located at `C:\databases\mydb`, including only user tables but both the name of the table and the name of its base table (if any), we run:
+
+```
+star list table -u -f tb C:\databases\mydb
+```
+
+###### `list index`
+
+```
+star list index [options] <db> <table>
+```
+
+The `list index` sub-command lists all indexes in a table, the name of which is specified by the `<table>` argument, in the database referenced by the absolute or relative path given in the `<db>` argument. The following options are available to configure the output of the commmand:
+
+Option        | Short form | Description
+------------- | ---------- | --------------------------------------------------------------------------------------------
+`--user`      | `-u`       | Includes user indexes in the output.
+`--system`    | `-s`       | Includes system indexes in the output.
+`--inherited` | `-i`       | Includes the indexes of base tables in the output.
+`--format`    | `-f`       | Lets you configure the formatting of the output by providing one or more formatting options.
+
+Formatting options:
+
+- `i` – The name of the index to list.
```

Comment:
Same applies here, as the comment I gave on `--format` on `list table`. It's the name of the current index being listed, not the one "to list".

Reply:
I think we're reading these sentences differently. To me they mean exactly what you say they should mean. Basically [The name of] [the index to list] where we were about to list an index, and here we're deciding whether or not to include the name of it. Regardless, I'm changing these sentences :)